pom.xml

```diff
@@ -38,10 +41,17 @@
     <dependency>
       <groupId>org.igniterealtime</groupId>
       <artifactId>tinder</artifactId>
+      <version>1.3.0</version>
```

Comment:
You appear to be overriding the version of this dependency, that before this change apparently was a managed dependency. Would it make sense to apply this change in the parent instead. Alternatively, perhaps consider removing the managed dependency?

Reply:
I intend to remove almost all of the managed versions in the parent (along with adding the sources/javadoc/signatures and upgrading to Java 8).
The managed versions are rather pointless: when I update it in the parent, I need to update the parent here in Jicofo, and I need to update the parent in e.g. Videobridge and then the Videobridge version here in Jicofo. If I miss to update the parent in Videobridge, there's a version mismatch anyway.